### ISSUE TITLE
Feature: Enable URL Key Specification

### ISSUE THREAD

**Issue by sonatard** [mirzemehdi/KMPNotifier#75]
We are sending keys like android_url from the server, so we want to make it possible to retrieve them.

https://github.com/mirzemehdi/KMPNotifier/blob/fe729de97830240ef2a1799959c15e13de53c25d/kmpnotifier/src/androidMain/kotlin/com/mmk/kmpnotifier/notification/AndroidNotifier.kt#L75

**Comment by mirzemehdi** [comment]
@sonatard can you please elaborate more what is the issue, couldn't get it?

**Comment by sonatard** [comment]
Our code is as follows. Since the key for retrieving urlData is different, we believe that using KMPNotifier results in the inability to retrieve urlData.

```kt
override fun onMessageReceived(remoteMessage: RemoteMessage) {
    val urlData = remoteMessage.data["android_open_url"]
```

https://github.com/mirzemehdi/KMPNotifier/blob/55b91898845eac05f1fc59cd6cc32fcc547c4a19/kmpnotifier/src/commonMain/kotlin/com/mmk/kmpnotifier/notification/Notifier.kt#L11

https://github.com/mirzemehdi/KMPNotifier/blob/fe729de97830240ef2a1799959c15e13de53c25d/kmpnotifier/src/androidMain/kotlin/com/mmk/kmpnotifier/notification/AndroidNotifier.kt#L75

**Comment by blackwiz4rd** [comment]
So basically the request is to use a custom key for the URL :)

**Comment by mirzemehdi** [comment]
So in your case, below should work, just add this to your initialization part
```kotlin

Notifier.KEY_URL= "android_open_url"

```

**Comment by mirzemehdi** [comment]
This is now available in `1.4.0` version
